deps/pylib.sh

```diff
@@ -1 +1,2 @@
 pip install requests_toolbelt filemagic ftputil
+pip install git+https://github.com/metabolite-atlas/metatlas
```

Comment:
Ok, next error is:
```
Traceback (most recent call last):
  File "/kb/deployment/pybin/trns_transform_mzML_LCMS_to_MetaboliteAtlas2_MAFileInfo.py", line 19, in <module>
    from metatlas import mzml_loader
  File "/kb/runtime/lib/python2.7/site-packages/metatlas/__init__.py", line 2, in <module>
    from .mzml_loader import mzml_to_hdf
  File "/kb/runtime/lib/python2.7/site-packages/metatlas/mzml_loader.py", line 9, in <module>
    import tables
ImportError: No module named tables
```
I think ```tables``` should be added into first line ```pip install ...```. But it's not enough. Because when I just run ```pip install tables``` I get such error:
```
Downloading/unpacking tables
  Downloading tables-3.2.0.tar.gz (7.0MB): 7.0MB downloaded
  Running setup.py (path:/tmp/pip_build_root/tables/setup.py) egg_info for package tables
    /usr/bin/ld: cannot find -lhdf5
    collect2: ld returned 1 exit status
    * Using Python 2.7.3 (default, Jan  9 2014, 18:36:02)
    * USE_PKGCONFIG: False
    .. ERROR:: Could not find a local HDF5 installation.
       You may need to explicitly state where your local HDF5 headers and
       library can be found by setting the ``HDF5_DIR`` environment
       variable or by using the ``--hdf5`` command-line option.
    Complete output from command python setup.py egg_info:
    /usr/bin/ld: cannot find -lhdf5
collect2: ld returned 1 exit status
* Using Python 2.7.3 (default, Jan  9 2014, 18:36:02)
* USE_PKGCONFIG: False
.. ERROR:: Could not find a local HDF5 installation.
   You may need to explicitly state where your local HDF5 headers and
   library can be found by setting the ``HDF5_DIR`` environment
   variable or by using the ``--hdf5`` command-line option.
```